src/integrationTest/resources/db/testmigration/V1_12__alter_audit_exception_table.sql

```diff
@@ -0,0 +1,7 @@
+
+ALTER TABLE dataload_schedular_audit  RENAME COLUMN scheduler_status TO status;
```

Comment:
can we try using flyways of main source and  just use other which needed for test data? we reduce code duplication